### ISSUE TITLE
[Bug]: Users cannot navigate properly inside the details list with ctrl+alt+arrow keys in edge, when onRenderHeader is defined in groupProps

### ISSUE THREAD

**Issue by RanaKh1** [microsoft/fluentui#23241]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Window 11, Browser- Edge
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/rana-khell/pen/dydZObV?editors=1010

## Actual Behavior
Turn on narrator, use ctrl + alt + arrow keys to navigate inside the details list.
The navigation is not right on the cells of the table, for example while being on "b", clicking ctrl+alt+right arrow key moves to the next row and column "blue".
![image](https://user-images.githubusercontent.com/86226959/170265692-e164b42c-1cf8-476f-a904-ec511c2cb391.png)

## Expected Behavior
The navigation with ctrl + alt + arrow keys should work properly, according to the direction of the arrow key.

I noticed that deleting "onRenderHeader: () => <span>header</span>" in the group props fixes the issue but this isn't our use case.

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by gouttierre** [comment]
@RanaKh1 - Thanks for filing this issue with us. I have added this to our accessibility queue for further investigations. To set expectations, the developers will review this issue once capacity allows.

cc @smhigley , @khmakoto
